src/utils/global-config.js

```diff
@@ -1,17 +1,29 @@
+const fs = require('fs')
+
 const Configstore = require('configstore')
 const { v4: uuidv4 } = require('uuid')
 
 const { getPathInHome } = require('../lib/settings')
 
+const CONFIG_NAME = 'netlify'
+
 const globalConfigDefaults = {
   /* disable stats from being sent to Netlify */
   telemetryDisabled: false,
   /* cliId */
   cliId: uuidv4(),
 }
 
+const legacyConfigPath = getPathInHome(['config.json'])
```

Comment:
We could look for the folder instead in case there are other files in `~/.netlify`, which would render this PR moot